static/styles/compose.scss

```diff
@@ -44,11 +44,20 @@
         font-weight: 400;
         line-height: 1em;
     }
-}
 
-#compose_buttons .button.small {
-    font-size: 1em;
-    padding: 3px 10px;
+    .button.small {
+        font-size: 1em;
+        padding: 3px 10px;
```

Comment:
I think we'll want the space around 
![image](https://user-images.githubusercontent.com/890911/61175390-6799f880-a563-11e9-8cc5-9db2029c73cb.png)

to be `3px 8px` instead of `3px 10px` (though the rest of the buttons should stay as 10px). 

Also the on-hover of "Reply" is a bit broken -- the right edge doesn't darken on hover. Not a blocker if we can't figure it out, but maybe worth spending a few minutes to see if there is a known solution.

Reply:
Made the change from `10px` -> `8px`.

The right border for the reply button was removed just below the quoted CSS here:
```
.compose_reply_button {
        border-radius: 4px 0px 0px 4px;
        border-right: none;
    }
```
I removed it to resemble the reply button from #12334.
On removing `border-right: none;` :

![reply_button_right_border](https://user-images.githubusercontent.com/25329595/61176313-56410280-a5dc-11e9-9968-8e832a89392a.gif)

Should I make this change?

Reply:
hm, the double-thickness line definitely isn't good. 
Can the second line be added only on-hover?

Reply:
I tried a few variations and it doesn't seem to work on hover.

Reply:
hm. Ok, we can deal with that as followup.